src/js/components/Pagination/StyledPageControl.js

```diff
@@ -30,8 +67,30 @@ const sizeStyle = props => {
 };
 
 export const StyledPaginationButton = styled(Button)`
-  padding: 4px;
-  border-radius: 4px;
+  ${props => {
+    const buttonStyle =
+      typeof props.kind === 'string'
+        ? buildButtonStyle(props.theme.button, props.kind)
+        : props.kind;
+    const adjustedPadding = calcAdjustedPadding(buttonStyle, defaultStyles.pad);
+    let buttonState = 'default';
+    if (props.active) buttonState = 'active';
+    else if (props.selected) buttonState = 'selected';
+    else if (props.disabled) buttonState = 'disabled';
+
+    return `
+      padding: ${adjustedPadding[buttonState].pad};
+      :hover {
+        padding: ${
+          buttonState === 'disabled'
```

Comment:
Can we just do a check here of !(buttonState === 'disabled') -- only reassign if button is enabled. otherwise, just let line 82 set the padding.

Reply:
Addressed.